### ISSUE TITLE
"Go To Page" Button for MOBILE BUTTONS

### ISSUE THREAD

**Issue by MixDatSalt** [bitfocus/companion#1499]
**Describe the feature**
Either allow an option to use a 15 button Stream deck emulator or preferably, make it so that the "set surface to page" action works with mobile buttons.

This issue is one of the only hurdles from this awesome platform being great on phones as well.

**Usecases**
"Again, this is for "Mobile Buttons" as it's already added for surface and emulator" (30 button emulator sucks on a 5 inch phone)

The user could create a "home page" with a bunch of buttons that go directly to different pages for different apps.

Maybe page "x" has a bunch of buttons dedicated to a specific app and instead of pressing the up arrow 15 times to get to that page, you could get there instantly. Then you could create a button on page "x" that goes back to the "home page" etc

**Comment by Julusian** [comment]
I think that this is a difficult one to resolve.

The problem is that the mobile buttons pages are not connecting in a way that lets core tell them to change page. 

tablet2 works around this by letting the user specify which buttons to assume are page change buttons.
tablet3 lets core tell it which are the special page buttons, but cant handle user designed buttons.
the emulator works by connecting as a registered device.

I think the solution here is to rethink the tablet and emulator pages.
The tablet page should be an 'infinite' scrolling list of buttons
The emulator page should be a single page view of the buttons, each instance can be its own device in the surfaces tab. Styled similarly (designed to be fullscreen on a tablet/phone)

Basically, split tablet3 into 2 pages, and make the cycle mode register as a fake device, and follow some of the special logic in the emulator page

**Comment by MixDatSalt** [comment]
Would it be possible to just add the ability to use the 15 key stream deck emulator? That way people could choose between 15 and XL and it would in theory do the exact same thing as the feature request (while having buttons big enough to push on phones as opposed to trying to use the XL).

 Then maybe someday down the road, it might be possible to do it with Mobile Buttons.

**Comment by stevespaw** [comment]
Related to this, is it possible have more than 1 emulator? I have that as a feature request.
Thanks

**Comment by Julusian** [comment]
@stevespaw not currently, but that is part of my plan here

**Comment by noiseboi** [comment]
I'd love to see this feature too especially as tablet3 buttons can now work in an almost kiosk mode. Alternatively, multiple emulators and a full screen mode for emulators.

Use case, non tech users on 8" Android tablets.

**Comment by Julusian** [comment]
Duplicate of https://github.com/bitfocus/companion/issues/1071
